### PR TITLE
Updating to version 0.5.2

### PR DESCRIPTION
**Pull request by cfcurtis**
I think I did this correctly, let me know if there is more required for updating versions.

I've also just finished automating release builds for Windows and MacOS, as well as publishing to PyPi. If there's a recommended procedure to add Flatpak builds to github actions, I'd be happy to do that as well.

One other thing - poking through the files I don't see anything about translations. I now have a setup.py script that handles compiling translation files, is there a way to switch the Flatpak build to use that?

### CODE REVIEW

File: com.github.cfcurtis.pdfstitcher.desktop [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-Version=1.0
+Version=0.5.2
```
Comment (hfiguiere) [inline com.github.cfcurtis.pdfstitcher.desktop:3]: Version here isn't the software version. It's the desktop file specification version.

it's the metainfo.xml you need to change.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.github.cfcurtis.pdfstitcher.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
             "sources": [
                 {
                   "type": "file",
-                  "url": "https://github.com/cfcurtis/pdfstitcher/archive/refs/tags/v0.5.zip",
+                  "url": "https://github.com/cfcurtis/pdfstitcher/archive/refs/tags/v0.5.2.zip",
                   "sha256": "77cd9f3d1e4559e8ea8d3c27fb36a7dcee8824d69d0f6457fb9b2a2c395909c8",
```
Comment (hfiguiere) [inline com.github.cfcurtis.pdfstitcher.json:58]: you also need to change the checksum to the new one.

**Comment by hfiguiere** [comment]
as stated you should update the metainfo.xml to update the version number.

**Comment by cfcurtis** [comment]
Up to v0.6.1 now, closing this one. Need a better workflow.

**Pull request closed without merging**